blocks/edit/da-library/da-library.js

```diff
@@ -88,11 +88,16 @@ class DaLibrary extends LitElement {
     if (e.key === 'Escape') closeLibrary();
   }
 
-  handleModalClose() {
+  handleModalClose(e) {
```

Comment:
@kmurugulla noticed this one small one that wasn't pre-existing. There's one below, too.

If you can fix, we can merge. I'll get the other linting issues fixed in another PR.